src/sdk/resources/personalDriveExceptions.ts

```diff
@@ -0,0 +1,9 @@
+import { IHttpClient } from '../http';
+import { PagedResource } from './resource';
+import { AccountMapException } from '../models/accountMaps';
+
+export class PersonalDriveExceptionsResource extends PagedResource<AccountMapException> {
+	constructor(httpClient: IHttpClient, personalDriveJobId: string) {
+		super(httpClient, 'item', 'item', 'item', 'item', `transfers/${personalDriveJobId}/personal_drives/exceptions`);
```

Comment:
@rklind and @avolinsky, I'm sure you're already talking but just in case - looks like this and @avolinsky's [PR are duplicates](https://github.com/portalarchitects/skysync-cli/blob/57cca3a638efd7b0d5145e52e09a53e3b6d40dd5/src/sdk/resources/personalDriveExceptions.ts#L7).

Reply:
yup, closed mine